### PR TITLE
Add home sidebar widget area template

### PR DESCRIPTION
**Pull request by jasonbraun**
Add page-home-sidebar.php template and widget model for users who want a sidebar home page with widgets.

### CODE REVIEW

File: lib/models/widget.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  		'description' => __( $description ),
+  		'before_widget' => '<div class="widget">',
+  		'after_widget' => '</div>',
+  		'before_title' => '<h3>',
+  		'after_title' => '</h3>'
+  	));
+
+  }
+
+  create_widget( 'Page Sidebar', 'page', 'Displays on the side of pages with a sidebar' );
```
Comment (atticoos) [inline lib/models/widget.php]: Been using a namespacing pattern, for example: https://github.com/SandersForPresident/SitesForBernie/blob/develop/lib/init.php

Can you add `namespace SandersForPresident\Wordpress\Widgets;` to the top of this file?

**Comment by atticoos** [comment]
We have a few code style standards set up on this project, which is why the build failed:

https://travis-ci.org/SandersForPresident/SitesForBernie/jobs/78939890#L236-L267

**Comment by jasonbraun** [comment]
Ah ok - I'll take care of this when I get back to a computer.

**Comment by atticoos** [comment]
No rush :ok_hand:

**Comment by jasonbraun** [comment]
I thought I had it after I fixed the spacing, but I keep getting this error, inline control structures not allowed, I think it's because I need to close with an else statement? But this is the way I've always done it. Let me know.

```
FILE: ...s/build/SandersForPresident/SitesForBernie/page-home-sidebar.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 22 | ERROR | [x] Inline control structures are not allowed
    |       |     (Generic.ControlStructures.InlineControlStructure.NotAllowed)
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
Time: 1.48 secs; Memory: 6.5Mb
```

**Comment by jasonbraun** [comment]
Alright I think I got it, build passes, yay!

**Comment by atticoos** [comment]
Awesome, thanks for cleaning up the build :smile: :+1:

**Pull request merged**